### PR TITLE
Add CVE page

### PR DESCRIPTION
**Pull request by martinkuba**
This is adding a page with a list of published security vulnerabilities across all repositories, similar to [Kubernetes docs](https://kubernetes.io/docs/reference/issues-security/official-cve-feed/).

### CODE REVIEW

File: content/en/docs/cve.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Common Vulnerabilities and Exposures
+weight: 102
+---
+
+This is a list of reported CVEs across all repositories in the OpenTelemetry org
+on GitHub. The raw data is stored in
+[this GitHub repository](https://github.com/open-telemetry/sig-security), and it
+is refreshed daily.
```
Comment (svrnm) [inline content/en/docs/cve.md:9]:
```suggestion
This is a list of reported Common Vulnerabilities and Exposures (CVEs) across all repositories in the [OpenTelemetry organisation on GitHub](https://github.com/open-telemetry/). The raw data is stored in
the [open-telemetry/sig-security](https://github.com/open-telemetry/sig-security) repository, and it
is refreshed daily.
```

File: content/en/docs/cve.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <th>Issue Summary</th>
+        <th>Severity</th>
+        <th>Repository</th>
+      </tr>
+    </thead>
+    <tbody>
+    </tbody>
+  </table>
+
+<script id="main-script">
```
Comment (svrnm) [inline content/en/docs/cve.md:24]: I guess, the purpose of this script is to make sure that the most up-to-date data is always live on the website? 

There is an alternative way: hugo supports the ability to read from JSON files (https://gohugo.io/functions/data/getjson/), so we could replace the javascript with a template function and the reload with a re-build of the website triggered on a new CVE being published? The **advantage** would be that we don't need javascript, which feels a little bit more accessible and also makes it easier for crawlers to collect this page (helping the CVEs to be findable)

We can start with the JS, but I would like to get @chalin's perspective when he's back

**Review by svrnm — DISMISSED** [review]
Looking forward to have this, I provided some initial feedback

File: content/en/docs/cve.md (hunk 2 above)
Comment (chalin) [reply content/en/docs/cve.md:23]: Yes, let's do build-time fetch of the data if y'all agree!

**Review by chalin — COMMENTED** [review]
I agree with both points that @svrnm raised: let's move this into the newly proposed `/security` section, and do data-fetch at build time.

File: content/en/docs/cve.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      <tr>
+        <th>CVE ID</th>
+        <th>Issue Summary</th>
+        <th>Severity</th>
+        <th>Repository</th>
+      </tr>
+    </thead>
+    <tbody>
+    </tbody>
+  </table>
```
Comment (chalin) [inline content/en/docs/cve.md:21]: Nit: unindent the HTML.

**Comment by chalin** [comment]
Otherwise, great job! LGTM

File: content/en/docs/cve.md (hunk 2 above)
Comment (martinkuba) [reply content/en/docs/cve.md:23]: How often is the site rebuilt? If we do this, should I remove the text about the list being updated daily?

Comment (chalin) [reply content/en/docs/cve.md:23]: As @svrnm suggested, let's go with the dynamic update for now. We can revisit the page in 2024 :).

**Review by theletterf — APPROVED** [review]

**Review by cartermp — APPROVED** [review]

**Pull request merged**